pkgs/servers/web-apps/discourse/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, makeWrapper
+, bundlerEnv
+, defaultGemConfig
+, v8
+}:
+
+let
+  rubyEnv = bundlerEnv {
+    name = "discourse-env";
+
+    # XXX: Gemfile.lock needs to be patched with due to bundix expecting bundler 1.x
+    # https://github.com/NixOS/nixpkgs/issues/68089
+    # perl -i -p0e 's/BUNDLED WITH.*2.*$/BUNDLED WITH\n   1.17.2/s' $out/Gemfile.lock
+
+    # afterwards generate gemset.nix from repo checkout using
+    # $ nix-shell -p bundix
+    # $ bundix
```

Comment:
Can these instructions be converted into shell script?:D

Reply:
Yes, that's the plan

Reply:
this is no longer necessary, right?